examples/common/pigweed/BUILD.gn

```diff
@@ -14,11 +14,34 @@
 
 import("//build_overrides/pigweed.gni")
 import("$dir_pw_build/target_types.gni")
+import("pigweed_rpcs.gni")
+
+if (enable_pw_rpc) {
+  import("//build_overrides/pigweed.gni")
+  import("$dir_pw_protobuf_compiler/proto.gni")
+}
 
 config("default_config") {
   include_dirs = [ ".." ]
 }
 
+if (enable_pw_rpc) {
```

Comment:
should we use a chip prefix like `chip_enable_pw_rpc` ? or is this a more global option.

Reply:
Done